gamedata/descriptions/levels/aliencodex.md

```diff
@@ -0,0 +1,6 @@
+Claim ownership of the Alien contract to complete this level.
+
+TODO
```

Comment:
You _might_ want to mention of the underhanded coding contest here (without linking to the winners 😄) to give a small nudge to the answer. Your call!

Reply:
Added the hint 😂